docs/pages/bare/using-expo-client.md

```diff
@@ -53,121 +73,67 @@ if (Constants.appOwnership === 'expo') {
-
-```js
-// App.js
-import * as React from 'react';
-import { StatusBar } from 'expo-status-bar';
-import { StyleSheet, Text, View } from 'react-native';
-import MapView from './MapView';
-
 export default function App() {
-  return (
-    <View style={styles.container}>
-      <Text style={{ fontSize: 20, marginBottom: 10, fontWeight: '600' }}>Behold, a map! ✨</Text>
-      <MapView
-        style={{
-          height: 300,
-          width: 300,
-          borderRadius: 20,
-          overflow: 'hidden',
-        }}
-      />
-      <StatusBar style="default" />
-    </View>
-  );
+  return <Blurhash blurhash="LGFFaXYk^6#M@-5c,1J5@[or[Q6." style={{ flex: 1 }} />;
 }
-
-const styles = StyleSheet.create({
-  container: {
-    flex: 1,
-    backgroundColor: '#fff',
-    justifyContent: 'center',
-    alignItems: 'center',
-  },
-});
 ```
 
-Problem solved! Now when we render the `<MapView />` component in the client, we'll fallback to a plain `View` placeholder. When we build the app in Xcode/Android Studio, we'll use the Mapbox map. You could also alternatively fallback to a map provided from `react-native-maps`, since it's included in the Expo SDK.
+</SnackInline>
+
+If you run this code in the Expo Go app you'll see the fallback view, if you were to build this locally with the [Expo CLI run commands](/workflow/expo-cli#compiling) then you'd see the native blur hash view.
 
-<ImageSpotlight alt="MapView working in an app built with Xcode and falling back to a placeholder in Expo Go" src="/static/images/expo-go-fallback.png" />
+## Native module detection
 
-As you may have already guessed, you can apply this concept for more than just third party dependencies. For example, maybe you wrote a native module to wrap your favorite analytics library and you want to provide a mock for it within the client.
+Native modules are added to the JavaScript global object by the runtime, this means you can conditionally check if they exist to ensure functionality:
 
 ```js
-// MyAnalytics.js
 import { NativeModules } from 'react-native';
 
-// Generic way of logging calls to MyAnalytics when module isn't available
-const makeLogger = name => options =>
-  console.log(`Called MyAnalytics.${name} with: ${JSON.stringify(options)}`);
+const isAvailable = !!NativeModules.MyAnalytics;
+```
 
-// Get native module or use fallback logger
-const MyAnalytics = NativeModules.MyAnalytics ?? {
-  logEvent: makeLogger('logEvent'),
-  setUser: makeLogger('setUser'),
-};
+This is nice because it ensures the native module _must_ be installed and linked. Two issues with this solution:
 
-// You usually want to wrap native function calls in functions on the JS side to
-// provide TypeScript typing and validation.
+1. You need to know the native module name ahead of time.
+2. You likely want an error to be thrown when a native module is missing in a custom build, this helps you determine if there was a native linking issue.
```

Comment:
```suggestion
2. You likely want an error to be thrown when a native module is missing in a custom build. This helps you determine if there is a native linking issue.
```

Reply:
Not a suggestion but a comment. This sounds like too technical (which is not a bad thing). So can we give an example or include an example on how to determine if there is a native linking issue.